lib/v2/screens/transfer/send/send_confirmation/components/generic_transaction_success_diaog.dart

```diff
@@ -0,0 +1,166 @@
+                          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                          children: <Widget>[
+                            Text(
+                              e.label!,
+                              style: Theme.of(context).textTheme.subtitle2OpacityEmphasis,
+                            ),
+                            Text(e.text.toString(), style: Theme.of(context).textTheme.subtitle2),
+                          ],
+                        ),
+                      ),
+                    )
+                    .toList(),
+              ],
+            ),
+            const SizedBox(height: 30.0),
+            Row(
+              children: [
+                Text('Date:  '.i18n, style: Theme.of(context).textTheme.subtitle2),
+                const SizedBox(width: 16),
+                Text(
+                  DateFormat('dd MMMM yyyy HH:mm').format(transaction.timestamp?.toLocal() ?? DateTime.now()),
+                  style: Theme.of(context).textTheme.subtitle2,
+                ),
+              ],
+            ),
+            Row(
+              children: [
+                Text('Transaction ID:  '.i18n, style: Theme.of(context).textTheme.subtitle2),
+                const SizedBox(width: 16),
+                Expanded(
+                  child: Text(
+                    transaction.transactionId ?? "",
+                    overflow: TextOverflow.ellipsis,
+                    style: Theme.of(context).textTheme.subtitle2,
+                  ),
+                ),
+                IconButton(
+                  icon: const Icon(Icons.copy),
+                  color: AppColors.lightGreen6,
+                  onPressed: () {
+                    Clipboard.setData(ClipboardData(text: transaction.transactionId ?? "No transaction ID")).then(
+                      (_) {
+                        ScaffoldMessenger.maybeOf(context)!
+                            .showSnackBar(SnackBar(content: Text("Copied".i18n), duration: const Duration(seconds: 1)));
+                      },
+                    );
+                  },
+                )
+              ],
+            ),
+            Row(
+              children: [
+                Text('Status:  '.i18n, style: Theme.of(context).textTheme.subtitle2),
+                const SizedBox(width: 16),
+                Container(
+                  decoration: const BoxDecoration(
+                      borderRadius: BorderRadius.all(Radius.circular(20)), color: AppColors.lightGreen6),
+                  child: Padding(
+                    padding: const EdgeInsets.only(top: 4, bottom: 4, right: 8, left: 8),
+                    child: Text(
+                      "Successful".i18n,
+                      overflow: TextOverflow.ellipsis,
+                      style: Theme.of(context).textTheme.subtitle2,
+                    ),
+                  ),
+                ),
+              ],
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
```

Comment:
DialogRow needs to be moved into its own file in another PR

Reply:
sure, but since this is all new code, writing this on a new file, would not affect the lines changed on this pr. 

Another way to do split this is. 

Open one PR just to create this dialog, and explain why is needed and when it will be used. 

Then  this entire dialog can be reviewed and tested in isolation.

Reply:
Ok but this code is used in 3 different places, it ended up in this file because I based it on another file that used the component. 

So we can split that component out, either way it's going to be another PR.

Reply:
Unused / removed